docs/learn/encyclopedia/errors-and-debugging/debugging-errors.mdx

```diff
@@ -55,8 +55,8 @@ The error is returned by the core immediately as a response to the transaction b
 
 | Error from Core | Explanation | Fix |
 | --- | --- | --- |
-| status: `TRY_AGAIN_LATER` | There is already a transaction from the same source account in memory. The transaction has been rejected due to too low fee and has been resubmitted too soon. The transaction has resource consumption higher than the network limits allow, e.g. it wants to use 200M instructions, while the ledger-wide limit is just 100M (the code is misleading here due to bug in Core’s error reporting). | Wait for the previous transaction to be applied and resubmit, or switch to channel accounts if higher throughput is needed. Wait more time before re-submitting the transaction. Build a transaction within the network limits. |
-| status: `ERROR`, error: `txMALFORMED` | Transaction is fundamentally wrong: Soroban operation is not the only operation in the transaction; Soroban extension is missing; Fee or resource fee is negative; Resource fee is greater than tx fee. Note: Most of these probably can be enhanced with diagnostic events and move to `txSOROBAN_INVALID` category. | Make sure the transaction is well-formed and the total transaction fee is high enough. |
+| status: `TRY_AGAIN_LATER` | There is already a transaction from the same source account in memory. The transaction has been rejected due to too low fee and has been resubmitted too soon. | Wait for the previous transaction to be applied and resubmit, or switch to channel accounts if higher throughput is needed. Wait more time before re-submitting the transaction. Build a transaction within the network limits. |
+| status: `ERROR`, error: `txMALFORMED` | Transaction is fundamentally wrong: Soroban operation is not the only operation in the transaction; Soroban extension is missing; Fee or resource fee is negative; Resource fee is greater than tx fee; Declared resources are higher than network-wide limits (e.g. it wants to use 200M instructions, while the ledger-wide limit is just 100M). Note: Most of these probably can be enhanced with diagnostic events and move to `txSOROBAN_INVALID` category. | Make sure the transaction is well-formed and the total transaction fee is high enough. |
```

Comment:
I don't think `Note: ...` sentence belongs to user-facing docs.

Reply:
removed it.